src/dcp/dcp.c

```diff
@@ -231,6 +234,23 @@ int main(int argc, char** argv)
                     }
                 }
                 break;
+            case 'e':
+                copy_xattrs_used = 1;
+                if (!strcmp(optarg,"none")) {
+                    mfu_copy_opts->copy_xattrs = XATTR_COPY_NONE;
+                } else if (!strcmp(optarg,"non-lustre")) {
```

Comment:
For code like this, I've found it more useful to have an array of { value, "string" } for all of the options (with an empty entry at the end), and then loop over the array looking for the matching string to get the value (possibly in a small helper function), or in the case of the usage message loop over the array and print all of the strings.  That avoids duplicating the string/value mapping a few times (as is done here and dsync.c), and avoids future errors if a new type is added but the usage message is not updated, or not updated in both programs, or similar.